packages/linter/src/index.ts

```diff
@@ -30,33 +30,34 @@ import { HeroImageIsDefined } from "./rules/HeroImageIsDefined";
 import { FastGoogleFontsDisplay } from "./rules/FastGoogleFontsDisplay";
 import { GoogleFontPreconnect } from "./rules/GoogleFontPreconnect";
 import { BoilerplateIsRemoved } from "./rules/BoilerplateIsRemoved";
-import { AmpImgUsesSrcSet } from './rules/AmpImgUsesSrcSet';
```

Comment:
same prettier comment as above (we actually have [singleQuotes](https://prettier.io/docs/en/options.html#quotes) as `true` [currently](https://github.com/ampproject/amp-toolbox/blob/cff4fc0f1c5548e1b39d15d737ad530ee22459dd/prettier.config.js#L19))

Reply:
er, actually running prettier on the file updated the quotes - are we just not including prettier in the precommit hook?

Reply:
looks like it, we need to fix that